internal/config/cgroupmanager/cgroupmanager.go

```diff
@@ -58,9 +58,9 @@ func SetCgroupManager(cgroupManager string) (CgroupManager, error) {
 	switch cgroupManager {
 	case systemdCgroupManager:
 		if node.CgroupIsV2() {
-			return new(Systemdv2Manager), nil
+			return new(SystemdV2Manager), nil
```

Comment:
this belongs to the first commit in the series, isn't it?

Reply:
yeah the commits are all over the place. I am inclined to leave it but could fix if you want.